PROJECT

```diff
@@ -0,0 +1,16 @@
+domain: slo.payu.com
+layout:
+- go.kubebuilder.io/v3
+projectName: argos
+repo: slo.payu.com/argos
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: slo.payu.com
+  group: argos
+  kind: Slo
+  path: slo.payu.com/argos/api/v1
+  version: v1
+version: "3"
```

Comment:
i need an explanation on this file. not sure what he does or it's propose

Reply:
Indeed no need, probably a leftover from an `init` flow

Reply:
actually when creating the API and groups locally with `kubebuilder`, it does need PROJECT file and you have it in redis-operator from same reason